src/app/services/faq.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { PostPass } from '../models/pass';
+import { ApiService } from './api.service';
+import { EventKeywords, EventObject, EventService } from './event.service';
+import { LoggerService } from './logger.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FaqService {
+  constructor(
+    private apiService: ApiService,
+    private eventService: EventService,
+    private loggerService: LoggerService
+  ) {
+  }
+
+  async getFaq() {
+    let res = null;
+    try {
+      this.loggerService.debug(`GETTING FAQ: }`);
+      res = await this.apiService.get('readfaq', { faq: 'faq' });
+      console.log(JSON.stringify(res))
+    } catch (e) {
+      this.loggerService.error(`${e}`);
+      this.eventService.setError(
+        new EventObject(
+          EventKeywords.ERROR,
+          e,
+          'Pass Service'
+        )
+      );
+      throw e;
+    }
+    return res[0].text;
```

Comment:
return from within the try catch. This is currently bombing when the api fails to get the faq object

Reply:
Addressed in: [Commit](https://github.com/bcgov/parks-reso-public/pull/313/commits/d9e72091727546caaf8d450f67098c7d4ad47ede)